### ISSUE TITLE
Ubuntu make errors - Multiple definition of pcre, undefined references to wx functions etc.

### ISSUE THREAD

**Issue by BRMatt** [etexteditor/e#69]
When trying to build e I get the [following error(s)](http://gist.github.com/328267).

I'm using the master branch, on ubuntu 9.1 and had to use the following patches to get this far:

http://gist.github.com/243464
http://e-texteditor.com/forum/viewtopic.php?t=3511

**Comment by ajpalkovic** [comment]
Yeah, I get the same problems.  I was able to get past all but one of the linker errors:

cxInternal.cpp:(.text+0x4d29): undefined reference to `eSettings::SetPageSettings(unsigned int, wxString const&, doc_id, int, int, wxString const&, std::vector<unsigned int, std::allocator<unsigned int> > const&, std::vector<cxBookmark, std::allocator<cxBookmark> > const&, eSettings::SubPage)'

That one I just haven't been able to get around.

**Comment by BRMatt** [comment]
Please can you share how you got past them?

**Comment by ajpalkovic** [comment]
Yeah, I'll try and clean up my changes and push them later this week.

**Comment by ajpalkovic** [comment]
http://www.e-texteditor.com/forum/viewtopic.php?p=14939#14939
